src/macros.rs

```diff
@@ -82,16 +82,11 @@ macro_rules! s {
         __item! {
             #[repr(C)]
             #[cfg_attr(feature = "extra_traits", derive(Debug, Eq, Hash, PartialEq))]
+            #[derive(Copy, Clone)]
             #[allow(deprecated)]
             $(#[$attr])*
             pub struct $i { $($field)* }
         }
-        #[allow(deprecated)]
-        impl ::Copy for $i {}
-        #[allow(deprecated)]
-        impl ::Clone for $i {
-            fn clone(&self) -> $i { *self }
-        }
```

Comment:
These changes are a nice cleanup, but would you mind moving them to a separate PR? To keep different types of changes atomic.